index.html

```diff
@@ -52,22 +52,28 @@
 			<div id="nino-leftSide" class="fsr">
 				<div layout="row" class="fw">
 					<header class="col-md-12 col-sm-6 col-xs-12">
-						<img src="images/avatar.jpg" alt="" class="avatar">
+						<img src="/home/roman/Pictures/IMG_9983.jpg" alt="" class="avatar">
```

Comment:
La ruta `/home/roman/Pictures/IMG_9983.jpg` solo la ves tu en ordenador, para que se vea en la web la imagen tiene que estar en el repositorio (o en algun sitio publico accesible). Yo subiria una foto tuya el repositorio a la carpeta `images` con el nombre que quieras (`avatar.jpg` por ejemplo) y volveria a escribir la ruta asi:
```suggestion
						<img src="images/avatar.jpg" alt="" class="avatar">
```